### PR TITLE
Update devops and delivery team docs

### PR DESCRIPTION
**Pull request by michaellzc**
Given the recent reorg, we should update the docs to make things less confusing.

#### Why not nest delivery docs under devops?

The handbook is already pretty nested, and the focus of delivery is still quite different from the cloud devops area, hence keeping them separate.

Close #1804

### CODE REVIEW

File: content/departments/product-engineering/engineering/cloud/delivery/index.md [hunk 1]
```diff
@@ -1,17 +1,14 @@
 # Delivery team
 
-The Delivery team (part of the [Enablement](../index.md) org).
+Delivery team is a part of the [DevOps](../devops/index.ms) team (prt of the [Cloud](../index.md) org).
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/delivery/index.md]:
```suggestion
Delivery team is a part of the [DevOps](../devops/index.ms) team (part of the [Cloud](../index.md) org).
```

File: content/departments/product-engineering/engineering/cloud/delivery/onboarding.md [hunk 2]
```diff
@@ -87,3 +90,5 @@ Welcome to the Delivery team! This document will guide you through delivery-spec
 
 - Develop any new material required to help others level up and understand our product
 - Add to the teams planning and OKRs
+
+[gcp]: https://console.cloud.google.com
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/delivery/onboarding.md:94]:
```suggestion
[GCP]: https://console.cloud.google.com
```

File: content/departments/product-engineering/engineering/cloud/devops/index.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Cloud DevOps Team
 
+Cloud DevOps team consists of two streams with different focus area.
+
+- DevOps
+- [Delivery]
+
+If you're looking DevOps stream documentation, stay on this page.
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/index.md]:
```suggestion
If you're looking for DevOps stream documentation, stay on this page.
```

File: content/departments/product-engineering/engineering/cloud/delivery/processes.md [hunk 4]
```diff
@@ -48,3 +50,9 @@ We still work in 2 week cycles, and have the following ceremonies:
    - This happens on the weeks we don't have planning, and is a check in on the plans and anything new
 3. Retro (biweekly)
    - A review of what we did for learing purposes
+
+### Issue tracking
+
+<!-- TODO: We should copy the content of google docs over here once the content is finalized -->
+
+The [Delivery GitHub project](https://github.com/orgs/sourcegraph/projects/205) is the single source of truth. You shoud go over this [doc](https://docs.google.com/document/d/1HBsAwXuuAVWMCl1_C2hNnKGmoOHGEmcpkQG_jQBSZeg/edit?usp=sharing) to understand how we use the board.
```
Comment (kevinwojo) [inline content/departments/product-engineering/engineering/cloud/delivery/processes.md]: We should probably make this document a page within the handbook

**Review by caugustus-sourcegraph — APPROVED** [review]
This PR means https://github.com/sourcegraph/handbook/pull/1804 can be closed out

File: content/departments/product-engineering/engineering/cloud/delivery/index.md [hunk 5]
```diff
@@ -1,17 +1,14 @@
 # Delivery team
 
-The Delivery team (part of the [Enablement](../index.md) org).
+Delivery team is a part of the [DevOps](../devops/index.ms) team (part of the [Cloud](../index.md) org).
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/delivery/index.md]:
```suggestion
Delivery team is a part of the [DevOps](../devops/index.md) team (part of the [Cloud](../index.md) org).
```

**Review by kevinwojo — APPROVED** [review]
Thanks for going the extra mile with the swim lane breakdown outta the gdoc

**Pull request merged**